src/components/clusterConfiguration/ReviewValidations.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import _ from 'lodash';
+import { Button, ButtonVariant } from '@patternfly/react-core';
+import { CheckCircleIcon, ExclamationTriangleIcon } from '@patternfly/react-icons';
+import {
+  global_success_color_100 as okColor,
+  global_warning_color_100 as warningColor,
+} from '@patternfly/react-tokens';
+import {
+  ValidationsInfo as ClusterValidationsInfo,
+  ValidationGroup as ClusterValidationGroup,
+  Validation as ClusterValidation,
+} from '../../types/clusters';
+import {
+  ValidationsInfo as HostValidationsInfo,
+  ValidationGroup as HostValidationGroup,
+  Validation as HostValidation,
+} from '../../types/hosts';
+import ClusterWizardContext from '../clusterWizard/ClusterWizardContext';
+import { ClusterWizardStepsType, findValidationFixStep } from '../clusterWizard/wizardTransition';
+import { wizardStepNames } from '../clusterWizard/ClusterWizardStep';
+import { Cluster, Host, stringToJSON } from '../../api';
+
+const AllValidationsPassed = () => (
+  <>
+    <CheckCircleIcon color={okColor.value} /> All validations passed
+  </>
+);
+
+const PendingValidations: React.FC<{ id: string; count: number }> = ({ id, count }) => (
+  <div id={id}>
+    There {count === 1 ? 'is' : 'are'} still {count} pending validation
+    {count === 1 ? '' : 's'}.
+  </div>
+);
+
+const ValidationActionLink: React.FC<{ step: ClusterWizardStepsType }> = ({ step }) => {
+  const { setCurrentStepId } = React.useContext(ClusterWizardContext);
+  return (
+    <Button variant={ButtonVariant.link} onClick={() => setCurrentStepId(step)} isInline>
+      {wizardStepNames[step]}
+    </Button>
+  );
+};
+
+const FailingValidation: React.FC<{
+  validation: HostValidation | ClusterValidation;
+  clusterGroup?: ClusterValidationGroup;
+  hostGroup?: HostValidationGroup;
+}> = ({ validation, clusterGroup, hostGroup }) => {
+  const issue = `Checking of ${_.lowerFirst(validation.message).replace(/\.$/, '')}, failed. `;
```

Comment:
Cc @nirfarkas

Reply:
We can tweak messages on the backend. But it would be good to agree on a generic structure to be used here.

Reply:
I think the warning icon should be replaced with an error icon 
![image](https://user-images.githubusercontent.com/65043359/108039364-abf55080-7044-11eb-836b-62792388b606.png)

Reply:
ok, I will update that in left-navigation as well